test/specs/scatter-plot.spec.js

```diff
@@ -0,0 +1,197 @@
+define(['d3', 'scatter-plot', 'scatterPlotDataBuilder'], function(d3, chart, dataBuilder) {
+    'use strict';
+
+    const aTestDataSet = () => new dataBuilder.ScatterPlotDataBuilder();
+    const buildDataSet = (dataSetName) => {
+        return aTestDataSet()
+            [dataSetName]()
+            .build();
+    };
+
+    describe('Scatter Plot', () => {
+
+        let scatterPlot, dataset, containerFixture, f;
+
+        beforeEach(() => {
+            dataset = buildDataSet('withFourNames');
+            scatterPlot = chart();
+
+            // DOM Fixture Setup
+            f = jasmine.getFixtures();
+            f.fixturesPath = 'base/test/fixtures/';
+            f.load('testContainer.html');
+
+            containerFixture = d3.select('.test-container');
+            containerFixture.datum(dataset).call(scatterPlot);
+        });
+
+        afterEach(() => {
+            containerFixture.remove();
+            f = jasmine.getFixtures();
+            f.cleanUp();
+            f.clearCache();
+        });
+
+        it('should render a chart with minimal requirements', () => {
+            expect(containerFixture.select('.scatter-plot').empty()).toBeFalsy();
```

Comment:
I really want us to move away from this format on the tests and start doing something like:
  let expected = 1;
  let actual = containerFixture.select('.scatter-plot').nodes().length;

  expect(actual).toEqual(expected);

No rush for it, as the rest are like this one...